setup.py

```diff
@@ -76,79 +76,6 @@ def __init__(self, dist):
         build_clib.__init__(self, dist)
         self.build_args = {}
 
-    def autotools_path(self):
```

Comment:
Eeek! Don't we need this stuff?

Reply:
I think it was needed only for readthedocs

Reply:
isn't it easier to ask people to install `automake` and `pkg-config` as requirements?

Reply:
No, this wasn't needed for readthedocs, it was needed for building wheels.

Reply:
ok, I'll revert that

Reply:
Actually, never mind, I think this is OK. I misread the diff.